pages/instructorScanPaper/instructorScanPaper.ejs

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <%- include('../partials/head'); %>
+    <link href="<%= asset_path('stylesheets/issues.css') %>" rel="stylesheet" />
+	<title></title>
+</head>
+<body>
+    <%- include('../partials/navbar', {navPage: 'news_item'}); %>
+
+    <div id="content" class="container">
+        <div class="card mb-4">
+            <div class="card-header bg-primary text-white">Scan a PDF Collection of Barcoded Scrap Paper Documents</div>
+
+            <div class="card-body scan-paper-description">
+                <p>Each page of the PDF collection that you are uploading should contain a barcode. The PDF should not be greater than 25MB. High-resolution, but compressed images, are recommended for each page of the PDF upload.</p>
+                <p>970 out of 1000 barcodes are currently decoded from PDF upload in testsuite and further optimization is needed. Feedback for optimization is requested. Please share scanning resolution and other related success/failures in #pl-dev to help in the ongoing development of this feature.</p>
+            </div>
+            <div class="card-body">
+
+                <form method="POST" name="scan-scrap-paper" method="POST" enctype="multipart/form-data">
+                        <input type="hidden" name="__action" value="scan_scrap_paper">
+                        <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+
+                        <div class="form-group">
+                            <label for="pdf-collection">Choose PDF file</label>
+                            <input class="form-control-file" id="pdf-collection" name="file" type="file" accept="application/pdf">
+                        </div>
+                        <button type="submit" class="btn btn-primary">Submit</button>
+                    </div>
+                </form>
+            </div>
+            </div>
+        </div>
```

Comment:
Indentation and nesting is all over the place here, and I think you have a few extraneous closing tags. Please patch this up.

Reply:
Done.